docs/runtime-upgrade.md

```diff
@@ -0,0 +1,58 @@
+# Testing a runtime upgrade in local
+This must be done before each new release we do in any of out runtimes.
+Supposing we are testing centrifuge runtime in local:
```

Comment:
given that the title of this document makes it clear that this is all about testing this in the local environment, you can just delete this line or at least remove the "supposing" to avoid confusion 🙏

Reply:
Ok 🤔  What I meant is that for the document purpose, the runtime chosen is "centrifuge", if the user wants to test other runtimes they should change some commands to make it works. 
Maybe I can write simply that 🤣

Reply:
Thinking it better, the sentence does not add any relevant information. The user can see in the command the chain used and change it. So finally I removed it.